control-plane/rest/service/src/v0/volumes.rs

```diff
@@ -13,9 +13,10 @@ async fn volume_share(
     protocol: NexusShareProtocol,
 ) -> Result<String, RestError<RestJsonError>> {
     let volume = MessageBus::get_volume(Filter::Volume(volume_id.clone())).await?;
+    assert!(volume.get_state().is_some());
 
     // TODO: For ANA we will want to share all nexuses not just the first.
-    match volume.children.first() {
+    match volume.get_state().unwrap().children.first() {
```

Comment:
I was wondering why we need the unwrap here and then it hit me, these share/unshare are now out of date, because we now have the volume operations in place, so we don't need to issue the operations against the nexuses themselves... we can just issue it directly to the volume_id.

Reply:
I'll look at this in a separate PR.